app/controllers/observations_controller/form_helpers.rb

```diff
@@ -16,20 +16,38 @@
 module ObservationsController::FormHelpers
   private
 
+  # Note this array syntax: The nested attributes MUST come last.
+  # VSCode won't allow nested square brackets (?).
   def permitted_observation_args
     [:place_name, :where, :lat, :long, :alt, :when, "when(1i)", "when(2i)",
-     "when(3i)", :notes, :specimen, :thumb_image_id, :is_collection_location,
-     :gps_hidden]
+     "when(3i)", :specimen, :thumb_image_id, :is_collection_location,
+     :gps_hidden, { notes: [:Other] }]
   end
 
   def update_permitted_observation_attributes
     @observation.attributes = permitted_observation_params || {}
   end
 
+  # NOTE: You MUST call `to_h` on the permitted params, because param nesting.
+  # As of rails 5, params are an ActionController::Parameters object,
+  # not a hash.
   def permitted_observation_params
     return unless params[:observation]
 
-    params[:observation].permit(permitted_observation_args)
+    params[:observation].permit(permitted_observation_args).to_h
+  end
+
+  # Symbolize keys; delete key/value pair if value blank
+  # Also avoids whitelisting issues
+  def notes_to_sym_and_compact
+    return Observation.no_notes unless notes_param_present?
+
+    symbolized = params[:observation][:notes].to_unsafe_h.symbolize_keys
+    symbolized.compact_blank!
+  end
+
+  def notes_param_present?
+    params[:observation] && params[:observation][:notes].present?
```

Comment:
```suggestion
    params.dig(:observation, :notes).present?
```

Reply:
@JoeCohen thanks — I'd forgotten about `dig`, good suggestion. 

MO has a helper method that does something similar, `param_lookup` in application_controller. Maybe a good idea to do a PR for a global replace for this method — although it is not applied as often as would be useful, `dig` fills the bill.

I may be retiring this PR, because I misunderstood the original issue. Nathan and Jason have sorted me out.